sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenterDeserializer.java

```diff
@@ -338,7 +353,9 @@ public boolean apply(Version version) {
       }
     });
     for (Version version : versions) {
-      result.add(version.toString());
+      String fromString = version.equals(low) ? low.getFromString() :
+                            version.equals(high) ? high.getFromString()  : "";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add parentheses to make the operator precedence explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS864)